app/views/admin/collections/show.html.erb

```diff
@@ -25,6 +25,14 @@ Unless required by applicable law or agreed to in writing, software distributed
     <%= button_tag('Edit Collection Info', class: 'btn btn-default', data: {toggle:"modal", target:"#new_collection"}) if can? :update, @collection %>
     <br/><br>
     Or <a href='https://wiki.dlib.indiana.edu/display/VarVideo/Avalon+Collections+Guide#AvalonCollectionsGuide-UploadingviaBatch'><b>learn how to</b></a> create items using batch upload.
+    <br /><br />
```

Comment:
not a fan of `<br>` for adding spacing....use CSS?

https://developer.mozilla.org/en/docs/Web/HTML/Element/br

Relevant quotes: 
`The HTML <br> element produces a line break in text (carriage-return). It is useful for writing a poem or an address, where the division of lines is significant.`

`Do not use <br> to increase the gap between lines of text; use the CSS margin property or the <p> element.`

Also in HTML5 it's just `<br>` in XHTML it's `<br />`